tests/Resources/Page/ODH/JupyterHub/JupyterLabSidebar.robot

```diff
@@ -19,3 +19,26 @@ Navigate To ${dir_path} In JupyterLab Sidebar
 
 ${file_obj} Is Visible In JupyterLab Sidebar
   Wait Until Element Is Visible  xpath://span[@class="jp-DirListing-itemText"][.="${file_obj}"]
+
+Get Install Plugin list from JupyterLab
+  ${plugin_names}    Create List
+  Maybe Open JupyterLab Sidebar      Extension Manager
+  Run Keyword And Return Status   Wait Until Page Contains Element       xpath://*[@class="jp-extensionmanager-disclaimer"]//*[contains(text(), "Enable")]    timeout=10
+  Run Keyword And Return Status         Click Button    Enable
+  Run Keyword And Return Status      Wait Until Page Contains Element     xpath://*[@class="jp-extensionmanager-disclaimer"]//*[contains(text(), "Disable")]     timeout=10
+  ${link_elements}=  Get WebElements  xpath://ul[@class="jp-extensionmanager-listview"]//li[contains(@class,"jp-extensionmanager-entry-ok")]//a
+  FOR  ${idx}  ${ext_link}  IN ENUMERATE  @{link_elements}  start=1
+        ${plugin_name}=  Get Element Attribute    ${ext_link}    text
+        ${href}=  Get Element Attribute    ${ext_link}    href
+        Append To List    ${plugin_names}    ${plugin_name}
+        ${status}=  Get HTTP Status Code   ${href}
```

Comment:
Do we need to check that the link is reachable for the plugin?

Reply:
I think it's a good-to-have check because users may try to use links. We could treat the unhealthy links as a "warnings" rather then a "failures".

Reply:
The problem is that even if the links are failing we can't do much about it, I don't think those links are defined by us

Reply:
well, at least we know and we can file an issue about it. Whoever has the accountability for that will handle it

Reply:
I have added the keyword for Warning and pushed it which is commented as of now. If we all agrees with the to replace with warning the i will remove/comment the Continue failure keyword.

Personally i feel we should keep it as it will give more insight information and will have good pointers to talk to dev team on the changes and  If it could be fix.That was the thought behind checking the status for each url.